### PR TITLE
FS3530: Update eligibility button action

### PR DESCRIPTION
**Pull request by NarenderRajuB**
Add eligibility button action

### CODE REVIEW

**Review by srh-sloan — COMMENTED** [review]
Couple of comments, again nothing major

File: app/helpers.py [hunk 1]
```diff
@@ -113,19 +117,22 @@ def format_rehydrate_payload(
...
         "returnUrl": returnUrl,
         "markAsCompleteComponent": markAsCompleteEnabled,
     }
-    formatted_data["questions"] = extract_subset_of_data_from_application(form_data, "questions")
+    formatted_data["questions"] = extract_subset_of_data_from_application(form_data, "questions") if form_data else None
     formatted_data["metadata"] = {}
     formatted_data["metadata"]["application_id"] = application_id
     formatted_data["metadata"]["form_session_identifier"] = application_id
     formatted_data["metadata"]["form_name"] = form_name
+    formatted_data["metadata"]["fund_name"] = fund_name
```
[the remark this answers is not in the document]
Comment (NarenderRajuB) [reply app/helpers.py:132]: We need this to pass to result page where we need this to pass to new application

File: app/default/eligibility_routes.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+from flask import render_template
+from flask import request
+from flask import url_for
+
+eligibility_bp = Blueprint("eligibility_routes", __name__, template_folder="templates")
+
+
+@eligibility_bp.route("/eligibility-result/<fund_name>/<round_name>", methods=["GET"])
+def eligiblity_result(fund_name, round_name):
+    current_app.logger.info(f"Eligibility launch result: {fund_name} {round_name}")
```
Comment (nuwan-samarasinghe) [inline app/default/eligibility_routes.py:17]: Can we have some comments to express each methods functionality

File: app/translations/cy/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -378,14 +378,36 @@ msgstr[3] "%(application_count)s ceisiadau"
...
+msgid "Back to your applications"
+msgstr "Yn ôl at eich ceisiadau"
+
+#: app/templates/eligibility_result.html:13
+msgid "Based on your answers, you can apply for "
+msgstr "Yn seiliedig ar eich atebion, gallwch wneud cais am"
+
+#: app/templates/eligibility_result.html:13
+#, python-format
+msgid "%(fund_title)s "
```
Comment (nuwan-samarasinghe) [inline app/translations/cy/LC_MESSAGES/messages.po:408]: Probably we might need to remove this or find a way to not to populate it here

**Review by nuwan-samarasinghe — CHANGES_REQUESTED** [review]

**Review by srh-sloan — APPROVED** [review]

**Review by nuwan-samarasinghe — APPROVED** [review]

**Pull request merged**